inventory/management/incoming/three_steps.rst

```diff
@@ -2,113 +2,110 @@
-:menuselection:`Configuration --> Locations` and select the one you want to rename.
-Change its name and hit save.
+To start the 3-steps reception process, create a *Request for Quotation* from the *Purchase App*, 
+add some storable products to it and confirm. Then, three pickings are created with your *Purchase
+Order* as the source document:
 
-Create a Sales Order
-====================
+1. The first one with a reference *IN* to designate the receipt
+   process;
 
-In the *Sales* app, create a quotation with storable products to
-deliver. Then, confirm it and three pickings will be created and linked
-to your sale order.
+2. The second one with a reference *INT*, which is the move to the
+   quality control zone;
+
+3. The last one with a reference *INT* to designate the move to stock.
 
 .. image:: media/three_steps_03.png
    :align: center
+   :alt: View of the three moves created by the purchase of products in three steps strategy
 
-Now, click on the button. You should see three different pickings:
-
--  The first one, with a reference PICK, designates the picking process;
+Process the receipt, quality control and entry in stock
+=======================================================
 
--  The second one, with a reference PACK, is for the packing process;
+As the receipt operation is the first one to be processed, it has a *Ready* status while the others 
+are *Waiting Another Operation*. The move quality becomes *Ready* as soon as the receipt one is 
+marked as done.
 
--  The third one, with a reference OUT, designates the shipping process.
+To access the receipt operation, click on the button from the *Purchase Order* or go back to the 
+*Inventory App* dashboard and click on the *Receipts* button.
 
 .. image:: media/three_steps_04.png
    :align: center
+   :alt: View of the button to press to see the receipts that need to be processed
 
-Process the picking, packing, and delivery
-==========================================
-
-The first operation to be processed is the picking and has a *Ready*
-status, while the others are *Waiting Another Operation*. The packing
-will become *Ready* as soon as the picking is marked as *Done*.
-
-You can enter the picking operation from here, or access it through the
-inventory app.
+In receipt order, the products are always considered available because they come from the supplier. 
+Then, the receipt can be validated.
 
 .. image:: media/three_steps_05.png
    :align: center
+   :alt: View of the move (internal transfer) from the reception area to the warehouse input area
 
-Note that, if you have the product in stock, it will be automatically
-reserved and you can validate the picking document.
+Once the receipt has been validated, the transfer to quality becomes *Ready*. Since the documents 
+are chained, the products which have been previously received are automatically reserved on the 
+transfer. Then, the transfer can be directly validated.
```

Comment:
```suggestion
Once the receipt has been validated, the transfer to quality becomes *Ready*. Because the documents 
are chained to each other, products previously received are automatically reserved on the 
transfer. Then, the transfer can be directly validated.
```
Suggestion: Use *since* and *while* to refer only to time, not to mean because or although (: